fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockEntityTypeTags.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.block.entity.BlockEntityType;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+public final class ConventionalBlockEntityTypeTags {
+	private ConventionalBlockEntityTypeTags() {
+	}
+
+	/**
+	 * Blocks should be included in this tag if their movement can cause serious issues such as world corruption
```

Comment:
Why do we need both a block tag and a block entity tag? (See below for design concerns)

Reply:
Because modders are adding to both block and block entity tags. One can have multiple blocks use the same block entity so maybe they want to disable all the blocks by block entity. And as cursed as it sounds, it looks like you can have the reverse as well where one block can return different block entities based on where it is in the world. So Blocks and Block Entities are not entirely 1-to-1 from what I can tell.

Currently, this is what I am seeing for tag usage.

Fabric Block Tags:
![image](https://github.com/FabricMC/fabric/assets/40846040/39427662-299b-4cd8-a453-a2063861e403)
![image](https://github.com/FabricMC/fabric/assets/40846040/3d59f22e-9340-4655-a3a4-f23ab00278d6)

Fabric Block Entity Tags:
None

Forge Block Tags:
![image](https://github.com/FabricMC/fabric/assets/40846040/559e2081-3d11-423c-b0d5-5e134a81deae)

Forge Block Entity Tags:
![image](https://github.com/FabricMC/fabric/assets/40846040/c11fda0e-19c9-4d60-b5c8-58564ca251f7)

So I added block entity tag to support however people might be setting up their block<->block entity relationships. If we do want remove the block entity tag, I'll have to find those block entity tag users on forge and verify first that their use cases can be handled by block tag without issue. Let me know if I should go ahead and verify if the block entity tag can safely be remove without harming modder's use cases

Reply:
@apple502j Good news! I reach out to 4 devs that I found using the Block Entity Type tag and all but one got back to me. The ones that did all said they are cool with switching to just the block tag so I went ahead and yeet the Block Entity Type tag entirely. Now there is only `c:relocation_not_supported` as a block tag. No Block Entity Type tag anymore. Should be cleaner.